docs/source/collect/collectd.md

```diff
@@ -0,0 +1,95 @@
+---
+title: CollectD
+description: Include collectd files
+---
+
+The `collectd` collector can be used to run a DaemonSet in the cluster with the parameters provided.
+The collector will delete and clean up this DaemonSet and any artifacts after it's created
+
+## Parameters
+
+In addition to the [shared collector properties](https://troubleshoot.sh/docs/collect/collectors/#shared-properties), the `collectd` collector accepts the following parameters:
+
+##### `namespace` (Optional)
+The namespace where the DaemonSet will be created.
+If not specified, it will assume the "current" namespace that the kubectl context is set to.
+
+##### `image` (Required)
+The image to use for the pods controlled by this DaemonSet.
+This should be accessible to the nodes in the cluster.
```

Comment:
We should probably list requirements for the image. Like what commands must be in the path.